CMakeLists.txt

```diff
@@ -669,6 +669,15 @@ add_custom_target(check-junit-AesGcmEager
 
     DEPENDS accp-jar tests-jar)
 
+add_custom_target(check-junit-DifferentTempDir
```

Comment:
Not convinced that this is a high signal test. There's no assertion anywhere that a different tmpdir was actually used. Just that these two tests succeeded when this property was set. I'm inclined to drop this test target.

Reply:
Since the PR already has the two approvals, I'll drop this in another PR.